common/lib/xmodule/xmodule/tests/__init__.py

```diff
@@ -48,10 +50,16 @@ class TestModuleSystem(ModuleSystem):  # pylint: disable=abstract-method
     ModuleSystem for testing
     """
     def __init__(self, **kwargs):
-        id_manager = CourseLocationManager(kwargs['course_id'])
+        course_id = kwargs['course_id']
+        id_manager = CourseLocationManager(course_id)
         kwargs.setdefault('id_reader', id_manager)
         kwargs.setdefault('id_generator', id_manager)
-        kwargs.setdefault('services', {}).setdefault('field-data', DictFieldData({}))
+
+        services = kwargs.get('services', {})
+        services.setdefault('cache', DoNothingCache())
```

Comment:
Would it be useful to have this wrapped by the `CacheService`?

Reply:
Resolved with https://github.com/edx/edx-platform/pull/29260/commits/1c78884946ff950072315add3247fc546eca04eb#diff-caad0b6dd6279f98a4b30df46f5bbd1b895e5167e188658de1d9ae308c06c560R59